### PR TITLE
Update rviz2 packages for bloom

### PR DESCRIPTION
**Pull request by nuclearsandwich**
This PR has grown to add addition content to support releasing rviz2 for ardent.

- the rviz directory is now ignored by ament and catkin, and the package.xml file has been renamed `package.disabled.xml`
- package.xml manifests have been added for vendor packages.
- version bump to 2.0.0 🎊 

---

_original body_

catkin_pkg checks that package versions match before bloom can overlay patches so this change needs to be made in source.

Alternatively, we could bump the versions of everything else up to match the true rviz version.

Right now the rviz debs are based on this branch and so haven't received updates from today's changes.

### CODE REVIEW

**Comment by wjwwood** [comment]
Wouldn't it be better to raise the version of the other packages?

Also, the `rviz` package is not to be packaged, as it's ROS 1 only (that's why I have an `AMENT_IGNORE` in there).

**Comment by nuclearsandwich** [comment]
> Wouldn't it be better to raise the version of the other packages?

yeah, I would be fine doing so, This was just the smallest  change I could make to start jobs.



> Also, the rviz package is not to be packaged, as it's ROS 1 only (that why I have an AMENT_IGNORE.

rviz is ignored by bloom with an `ardent.ignored` file but it would seem that catkin_pkg parses the directory tree before filtering out ignored packages.

```
steven@steven-workstation:~/o/r/r/rviz-release ❯ cat ardent.ignored 
rviz
steven@steven-workstation:~/o/r/r/rviz-release ❯ git bloom-release ardent
+++ Cloning working copy for safety
Processing release track settings for 'ardent'
Checking upstream devel branch for package.xml(s)
Cloning into '/tmp/tmpq1_pobpf/upstream'...
remote: Counting objects: 18421, done.
remote: Compressing objects: 100% (12/12), done.
remote: Total 18421 (delta 1), reused 3 (delta 1), pack-reused 18408
Receiving objects: 100% (18421/18421), 11.23 MiB | 1.52 MiB/s, done.
Resolving deltas: 100% (13296/13296), done.
Checking connectivity... done.
Looking for packages in 'ros2' branch... found 6 packages.
Traceback (most recent call last):
  File "/home/steven/osrf/bloom-ros2.venv/bin/git-bloom-release", line 9, in <module>
    load_entry_point('bloom', 'console_scripts', 'git-bloom-release')()
  File "/home/steven/osrf/bloom2env/bloom/bloom/commands/git/release.py", line 353, in main
    args.unsafe)
  File "/home/steven/osrf/bloom2env/bloom/bloom/commands/git/release.py", line 247, in execute_track
    settings = process_track_settings(track_dict, release_inc)
  File "/home/steven/osrf/bloom2env/bloom/bloom/commands/git/release.py", line 192, in process_track_settings
    track_dict['ros_distro'])
  File "/home/steven/osrf/bloom2env/bloom/bloom/commands/git/release.py", line 145, in find_version_from_upstream
    meta = get_upstream_meta(upstream_repo.get_path(), ros_distro)
  File "/home/steven/osrf/bloom2env/bloom/bloom/commands/git/release.py", line 120, in get_upstream_meta
    name, version, packages = get_package_data(current_branch, quiet=False, release_directory=directory)
  File "/home/steven/osrf/bloom2env/bloom/bloom/packages.py", line 91, in get_package_data
    version = verify_equal_package_versions(packages.values())
  File "/home/steven/osrf/bloom2env/catkin_pkg/src/catkin_pkg/packages.py", line 176, in verify_equal_package_versions
    raise RuntimeError('Two packages have different version numbers (%s != %s):\n- %s\n- %s' % (package.version, version, package.filename, packages[0].filename))
RuntimeError: Two packages have different version numbers (1.12.11 != 0.0.1):
- /tmp/tmpq1_pobpf/upstream/rviz/package.xml
- /tmp/tmpq1_pobpf/upstream/rviz2/package.xml
```

**Comment by wjwwood** [comment]
We could try a `CATKIN_IGNORE` file as well.

**Comment by nuclearsandwich** [comment]
One question about licensing info in the vendor packages, since at least the ogre, and yamlcpp packages will be distributing the built version of the upstream software, should they list the upstream license instead of or in addition to the Apache license?

- assimp MIT
- ogre BSD
- yaml-cpp MIT

**Comment by wjwwood** [comment]
@nuclearsandwich That's a good point, I think it should in addition to, as I feel the `Apache 2.0` is to cover the cmake code and other stuff in those vendor packages.

**Comment by wjwwood** [comment]
Done.

**Comment by nuclearsandwich** [comment]
cool, I've got another change to ogre's deps. For the build with opengl to succeed we also need xrandr. I'll add that now.

File: rviz_assimp_vendor/package.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
   <license>Apache License 2.0</license>  <!-- the contents of this package are Apache 2.0 -->
   <license>BSD</license>  <!-- assimp is BSD (http://assimp.sourceforge.net/main_license.html) -->
 
-  <url type="website">http://assimp.sourceforge.net/index.html<url/>
+  <url type="website">http://assimp.sourceforge.net/index.html</url>
```
Comment (wjwwood) [inline rviz_assimp_vendor/package.xml]: I should sleep more, lol 🦀 

Sorry about that.

**Comment by nuclearsandwich** [comment]
I think this is ready to merge.

**Pull request merged**